Kickstarter-iOS/Views/Controllers/ProjectPamphletViewController.swift

```diff
@@ -17,6 +17,31 @@ public final class ProjectPamphletViewController: UIViewController {
 
   @IBOutlet weak private var navBarTopConstraint: NSLayoutConstraint!
 
+  private let backThisProjectContainerViewMargins = Styles.grid(3)
+  private let backThisProjectContainerView: UIView = {
+    return UIView(frame: .zero) |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  private let backThisProjectButton: UIButton = {
+     return UIButton(type: .custom) |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  private var backThisProjectContainerSublayer: CAShapeLayer {
```

Comment:
I'm not sure if you were going to do some changes to this logic that we discussed on the call with Justin, but I'm starting to think this should be a `function` that takes a `frame` (`backThisProjectContainerView.bounds`), maybe even `radii` with default value of 16. This way it will be more reusable ¯\_(ツ)_/¯

Reply:
@dusi sounds like a great idea for a future refactor if we end up wanting to reuse this layer!